modules/slugs/includes/class-qtranslate-slug.php

```diff
@@ -471,6 +464,19 @@ function filter_request( $query ) {
                     $function             = 'get_term_link';
                 }
             endforeach;
```

Comment:
Why are the taxonomies in the main "if" inside the block for the custom type? This looks wrong. It wasn't so in the origin plugin.

Reply:
If none of the conditions above are matched, custom post types first and taxonomies later are tried.
Original plugin tried those for every query, which was not needed.

Reply:
OK